sys/net/network_layer/ng_ipv6/hdr/ng_ipv6_hdr.c

```diff
@@ -63,6 +63,9 @@ ng_pktsnip_t *ng_ipv6_hdr_build(ng_pktsnip_t *payload,
 #endif
         memcpy(&hdr->src, src, src_len);
     }
+    else {
+        ng_ipv6_addr_set_unspecified(&hdr->src);
```

Comment:
A little bit nitpicky but can you add the line

```C
DEBUG("ipv6_hdr: set packet destination to ::\n");
```

to stay consistent with the other case.